modules/meb_api/lib/dgi/automation/service.rb

```diff
@@ -14,6 +14,8 @@ class Service < MebApi::DGI::Service
         STATSD_KEY_PREFIX = 'api.dgi.automation'
 
         def get_claimant_info(type = 'Chapter33')
+          type ||= 'Chapter33'
```

Comment:
If type is nil then it will default to 'Chapter33', so this line and the others like it can be dropped.